BetterTogetherApp/templates/BetterTogetherApp/share_food_create.html

```diff
@@ -1,12 +1,48 @@
-<h1>Create new ShareFood event</h1>
-<form action='{% url "BetterTogetherApp:create_share_food" %}' method="post">
-    {% csrf_token %}
-    Location Name: {{ form.location_name }}
-    <br> Location: {{ form.location }}
-    <br> Description: {{ form.description }}
-    <br> Number of Participants: {{ form.num_people }}
-    <br><h3>Select the time to meet with the others: </h3>{{ date_time }}
-    <button type="submit">Submit</button>
-</form>  
-
-<a href="{% url 'BetterTogetherApp:share_food_index1' %}"> Go BACK to index</a>
\ No newline at end of file
+{% load static %}
+<html>
+
+<head>
+    <title> Create ShareFood Event </title>
+    <link href="{% static "css/sharefood-create-style.css" %}" rel="stylesheet">
+    <link rel="icon" href="{% static "image/puzzle.png" %}">
+    <link href="https://fonts.googleapis.com/css?family=Hind|Lobster|Roboto&display=swap" rel="stylesheet">
```

Comment:
You should put an import font in the css file

Reply:
already fixed it ,thanks